source/speech/extensions.py

```diff
@@ -22,6 +22,14 @@
 Handlers are called without arguments.
 """
 
+speechPaused = Action()
```

Comment:
```suggestion
post_speechPaused = Action()
```

From codingStandards: https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md#identifier-names